strings/src/main/java/com/revenuecat/purchases/strings/ReceiptStrings.kt

```diff
@@ -0,0 +1,10 @@
+package com.revenuecat.purchases.strings
+
+object ReceiptStrings {
+    const val DATE_STALE_BACKGROUND = "Checking if cache is stale AppInBackground %s"
+    const val QUERYPURCHASES_CLEAN_PREV_SET_HASHED_TOKEN = "[QueryPurchases] Cleaning previously sent tokens"
```

Comment:
should `[QueryPurchases]` be moved out of here and reused? maybe in a log intent?

Reply:
@vegaro made a comment in #223, copying comment here: "Interesting we only do this [QueryPurchases] thing here, where we say what the log is about. I think it's such an implementation detail that we can probably remove it" 

I ended up removing `[QueryPurchases]` from all string files.